geometry_msgs/Transform search_axis
float32 eps_degrees_tolerance
float32 search_box_dimension
float32 min_height
---
bool success